app.py

```diff
@@ -6,6 +7,7 @@
 import typing
 from dataclasses import dataclass
 
+from datetime import datetime, timezone
```

Comment:
lets import the whole thing instead
```suggestion
import datetime
```

Reply:
resolved